src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partnerStatus;
+
+  if (isMarried) {
+    partnerStatus = sex === 'm' ? 'wife' : 'husband';
+  } else {
+    partnerStatus = 'not married';
+  }
```

Comment:
```suggestion
  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
```

can be a bit simplify